terraform-google-modules/terraform-example-foundation
# Add 1 org tests

Adding kitchen test for step `1-org` 

Tests added:

- check common folder creation
- check projects creation
- check service enabling
- check organization policies
- check folder organization policies
- check log sinks
- check BigQuery datasets
- check SCC notification
- check pub/sub topic
- check pub/sub subcription


Code Fixes:

- Add `include_children = true` to log sink creation
- Fixes `constraints/compute.disableSerialPortAccess` constraints name
